### PR TITLE
feat: set updates

### PR DESCRIPTION
**Pull request by pyphilia**

### CODE REVIEW

File: src/i18n/langs/en.json [hunk 1]
```diff
@@ -28,8 +27,22 @@
...
+  "GLOBAL_MODEL_TITLE": "Global Model",
+  "GLOBAL_MODEL_DESCRIPTION": "Radiative balance with greenhouse gases, clouds and surface",
   "Based on climate education research • Adapted for middle and high schools • Tested in classrooms • Easily embedded in Graasp activities and other web platforms": "Based on climate education research • Adapted for middle and high schools • Tested in classrooms • Easily embedded in Graasp activities and other web platforms",
-  "Brief tutorial of the simulations": "Brief tutorial of the simulations"
+  "Brief tutorial of the simulations": "Brief tutorial of the simulations",
+  "CHEMISTRY_SIMULATIONS_TITLE": "Chemistry Simulations",
+  "SEND_EMAIL_BUTTON": "Send us an email",
+  "CONTACT_FORM_TEXT": "We'll be more than happy to hear about your suggestion or feedback regarding your use of the simulations. It will help for further improvements. Write an email to us, we will reply in a few days.",
+  "TEAM_DESCRIPTION": "Graasp Climate is a collaboration between teachers from various Swiss secondary schools, the Graasp association and scientific and educational advisors.",
```
Comment (hagoptaminian) [inline src/i18n/langs/en.json:37]: "Graasp Climate is a collaboration between teachers from various Swiss secondary schools, the Graasp **Association,** and scientific and educational advisors."

File: src/i18n/langs/en.json [hunk 2]
```diff
@@ -28,8 +27,22 @@
...
+  "TEAM_ACKNOLEDGEMENTS": "We are also grateful to the researchers who helped with their valuable inputs (Prof. D. Archer, U Colorado, Prof. A. Ohmura, ETHZ, Prof. Natalie Krivova, MPS, Prof. Michael Petterson, U Auckland).",
```
Comment (hagoptaminian) [inline src/i18n/langs/en.json:38]: "We are also grateful to the following researchers who helped with their valuable inputs: Prof. D. Archer (U Colorado), Prof. A. Ohmura (ETHZ), Prof. Natalie Krivova (MPS), and Prof. Michael Petterson (U Auckland)."

File: src/components/subpages/team/Team.astro [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+              img={TG}
+              name="Thierry Gerez"
+              roles={[t('CHEMISTRY_TEACHER_MALE')]}
+            />
+          </div>
+          <div class="col col-sm-4 col-lg-3 col-xl-2">
+            <TeamMember
+              img={BM}
+              name="Béatrice Mandanis"
+              roles={[t('CHEMISTRY_TEACHER_FEMALE'), 'stepha']}
```
Comment (hagoptaminian) [inline src/components/subpages/team/Team.astro:86]: I believe 'stepha' here is an error

File: src/i18n/langs/en.json [hunk 4]
```diff
@@ -28,8 +27,22 @@
...
-  "Energy balance": "Energy balance",
-  "Global model including radiation, greenhouse gases, clouds and ice": "Global model including radiation, greenhouse gases, clouds and ice",
+  "GLOBAL_MODEL_TITLE": "Global Model",
+  "GLOBAL_MODEL_DESCRIPTION": "Radiative balance with greenhouse gases, clouds and surface",
   "Based on climate education research • Adapted for middle and high schools • Tested in classrooms • Easily embedded in Graasp activities and other web platforms": "Based on climate education research • Adapted for middle and high schools • Tested in classrooms • Easily embedded in Graasp activities and other web platforms",
-  "Brief tutorial of the simulations": "Brief tutorial of the simulations"
+  "Brief tutorial of the simulations": "Brief tutorial of the simulations",
+  "CHEMISTRY_SIMULATIONS_TITLE": "Chemistry Simulations",
+  "SEND_EMAIL_BUTTON": "Send us an email",
+  "CONTACT_FORM_TEXT": "We'll be more than happy to hear about your suggestion or feedback regarding your use of the simulations. It will help for further improvements. Write an email to us, we will reply in a few days.",
```
Comment (hagoptaminian) [inline src/i18n/langs/en.json:36]: We would love to hear your suggestions, comments, and feedback on the simulations.

File: src/i18n/langs/fr.json [hunk 5]
```diff
@@ -29,7 +28,21 @@
   "Copyright": "Copyright",
   "Page Not Found": "Page non trouvée",
   "Approved by the International Year of Basic Sciences for Sustainable Development 2022": "Approuvé par l'Année internationale des sciences fondamentales pour le développement durable 2022",
-  "Energy balance": "Bilan énergétique",
-  "Global model including radiation, greenhouse gases, clouds and ice": "Modèle global incluant le rayonnement, les gaz à effet de serre, les nuages et les glaces",
-  "Brief tutorial of the simulations": "Bref tutoriel des simulations"
+  "GLOBAL_MODEL_TITLE": "Bilan énergétique",
```
Comment (hagoptaminian) [inline src/i18n/langs/fr.json:31]: Should this be updated to the French equivalent of 'Global Model'?

**Review by hagoptaminian — CHANGES_REQUESTED** [review]
Hi @pyphilia this looks really nice. I left some comments mostly on (English) language improvements and a few small errors. I think the resolution/dimensions of education21 logo at the bottom are a bit off; maybe make it a bit smaller?

File: src/components/subpages/team/Team.astro (hunk 3 above)
Comment (pyphilia) [reply src/components/subpages/team/Team.astro:86]: it is correct! it's the real name of the school, at least what is written in Philippe's school.

File: src/i18n/langs/fr.json (hunk 5 above)
Comment (pyphilia) [reply src/i18n/langs/fr.json:31]: I think it's supposed to be the translation.

**Comment by pyphilia** [comment]
Thank you for the review! Could you provide a screenshot of the logo your mentioning? I think it has exactly the same dimensions as the others, but it has more margins within the image so it appears smaller.

**Comment by hagoptaminian** [comment]
Hi @pyphilia 

- Screenshot of Education21 logo stretching out:

Screen Shot 2024-09-11 at 11 09 08 AM

- About the 'stepha' under Béatrice's name: Maybe double-check with Philippe. I did some googling and couldn't find a school with that name. What makes me suspect that it's an error is that the next name in the list is 'Stéphane', so maybe just a typo. But I'll take your word for it.

**Comment by pyphilia** [comment]
Which browser are you using? I don't see this effect
Screenshot 2024-11-20 at 12 01 43

And indeed, I think it was a typo, I changed the school name.

**Pull request merged**